src/llnode.cc

```diff
@@ -491,6 +522,7 @@ bool PluginInitialize(SBDebugger d) {
       " * -n, --name  name     - all properties with the specified name\n"
       " * -s, --string string  - all properties that refer to the specified "
       "JavaScript string value\n"
+      " * -r, --recursive      - walk through references tree recursively\n"
```

Comment:
`-r` is incompatible with `-s` and `-n`, right? If so this should be documented, otherwise, we should have tests for `-r -s <string>` and `-r -n <property>`.